pom.xml

```diff
@@ -18,8 +18,8 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>3.6.0</version>
                 <configuration>
-                    <source>1.8</source>
-                    <target>1.8</target>
+                    <source>11</source>
+                    <target>11</target>
```

Comment:
This "could" be left at 8 but I updated it to 11 since [Jetty 10+ requires Java 11+](https://www.eclipse.org/jetty/download.php#version-table).